.github/workflows/convention-develocity-shared-verification.yml

```diff
@@ -0,0 +1,123 @@
+
+on:
+  push:
+    branches: [ main ]
+    paths: [ 'convention-develocity-shared/**' ]
+  pull_request:
+    branches: [ main ]
+    paths: [ 'convention-develocity-shared/**' ]
+  workflow_dispatch:
+
+jobs:
+  build:
+    name: Build Convention Develocity Shared
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up JDK 8
+        uses: actions/setup-java@v4
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+      - name: Set up Gradle
+        uses: gradle/actions/setup-gradle@v3
+      - name: Build with Gradle
+        working-directory: convention-develocity-shared
+        run: ./gradlew build publishToMavenLocal
+      - name: Upload convention
+        uses: actions/upload-artifact@v4
+        with:
+          name: convention-develocity-shared
+          path: ~/.m2/repository/com/myorg
+
+  verification_gradle:
+    name: Verify Example Build for Gradle ${{ matrix.versions.version }}
+    needs: [build]
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        versions:
+          - sample: '5'
+            version: '5.0'
+          - sample: '6'
+            version: '6.0.1'
+          - sample: '6.9_and_later'
+            version: '6.9.4'
+          - sample: '6.9_and_later'
+            version: '7.0.2'
+          - sample: '6.9_and_later'
+            version: '8.0.2'
+          - sample: '6.9_and_later'
+            version: '(Current)'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Set up JDK 8
+        uses: actions/setup-java@v4
+        with:
+          java-version: '8'
+          distribution: 'temurin'
+      - name: Set up Gradle
+        uses: gradle/actions/setup-gradle@v3
+      - name: Download plugin
+        uses: actions/download-artifact@v4
+        with:
+          name: convention-develocity-shared
+          path: ~/.m2/repository/com/myorg
+      - name: Set Gradle version
+        if: ${{ matrix.versions.version != '(Current)' }}
+        working-directory: convention-develocity-shared/examples/gradle_${{ matrix.versions.sample }}
+        run: |
+          sed -i '/distributionSha256Sum.*/d' gradle/wrapper/gradle-wrapper.properties
+          ./gradlew wrapper --gradle-version=${{ matrix.versions.version }} --no-scan
+          ./gradlew wrapper --gradle-version=${{ matrix.versions.version }} --no-scan
+      - name: Verify example build
+        id: build
+        working-directory: convention-develocity-shared/examples/gradle_${{ matrix.versions.sample }}
+        run: ./gradlew build -Ddevelocity.url=https://ge.solutions-team.gradle.com
+        env:
+          DEVELOCITY_ACCESS_KEY: ${{ secrets.DV_SOLUTIONS_ACCESS_KEY }}
+      - name: Verify Build Scan published
+        if: ${{ !steps.build.outputs.build-scan-url }}
+        run: echo "::error ::No Build Scan published"; exit 1
+
+  verification:
```

Comment:
maybe should be `verification_maven` to match `verification_gradle` above

Reply:
Yes, it should be. Good catch.

Reply:
Fixed in 230d39a99170dfbab692f4812392e3ef51c7ca4f